src/Umbraco.Web.UI.Client/src/packages/core/content/workspace/views/edit/content-editor.element.ts

```diff
@@ -16,8 +16,6 @@ import './content-editor-tab.element.js';
 
 @customElement('umb-content-workspace-view-edit')
 export class UmbContentWorkspaceViewEditElement extends UmbLitElement implements UmbWorkspaceViewElement {
-	//@state()
```

Comment:
This is very close to become a feature, so I will leave these out commented lines here. It only because its a new feature to enable properties without groups/tabs that I haven't implemented this fully yet.

Reply:
I've added it back in with a proper comment